browser-sign-in/OktaBrowserSignIn/WelcomeViewController.swift

```diff
@@ -58,6 +99,12 @@ final class WelcomeViewController: UIViewController {
             }
         }
     }
+
+    @IBAction private func signInTapped() {
+        auth?.ephemeralSession = ephemeralSwitch.isOn
+        let isBiometricsEnabled = biometricStorageSwitch.isOn
+        isBiometricsEnabled ? SignInWithBiometrics() : signIn()
```

Comment:
It feels weird using an inline conditional like this to perform an action. Usually you'd use this in assigning a value to a variable or a property, but triggering another function doesn't feel straightforward.

Instead I'd suggest putting this into an actual `if/else` block, to make it clear that a decision is being made.

It might also be good to put some comments in for a developer to read, letting them know why the decision is being made.  A developer would only choose one or the other approach, so it'll be important for us to call this out, so they don't think they need to implement code like this in their own apps.

Reply:
I agree. I've updated this.  the ternary operator could also serve as an if/else expression however I agree with your comment. Its better to be explicit and use the if/else block